tests/test_integration.py

```diff
@@ -580,7 +583,7 @@ def test_info(self):
 class MigrationTest(IGVMTest):
     def setUp(self):
         super(MigrationTest, self).setUp()
-        vm_build(VM_HOSTNAME)
+        vm_build(VM_HOSTNAME, rebuild=True)
```

Comment:
Is this still needed since we clean cert before each test?

Reply:
Probably not. Will remove